runsc/container/container.go

```diff
@@ -311,6 +311,10 @@ func New(conf *boot.Config, args Args) (*Container, error) {
 	if isRoot(args.Spec) {
 		log.Debugf("Creating new sandbox for container %q", args.ID)
 
+		if args.Spec.Linux != nil && args.Spec.Linux.CgroupsPath == "" {
+			args.Spec.Linux.CgroupsPath = "/" + args.ID
+		}
+
```

Comment:
After `cgroup.New`, add a condition to ignore the error if equal to `EACCES` and `conf.Rootless` is true, logging a warning that cgroup configuration was skipped.

Reply:
I guess, `cgroup.New` doesn't fail because it doesn't attempt to create anything yet. Unless it fails on reading `/proc/self/cgroups`?
Tests fail for me on `cg.Install` down below with `EROFS` at `mkdir` for each cgroup controller. But in tests `conf.Rootless == false`. Any other bit too look for in order to differentiate when it's OK to skip cgroups?

Reply:
Yes, `cgroup.Install` is where it will fail. Tests set `--TESTONLY-unsafe-nonroot` which maps to `config.TestOnlyAllowRunAsCurrentUserWithoutChroot`.

Reply:
OK. So I added exclusions for this test mode and rootless mode.

Reply:
Looks great. Thanks!